bin/build-prod.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+SERVER_NAME=${1:-docker.io/lmorchard/idea-town:latest}
```

Comment:
It'd probably be good to use a different default server name, or just spit out a help message and halt if one isn't set